README.md

```diff
@@ -93,6 +93,10 @@ const sqsConsumer = SqsConsumer.create({
     handleMessage: async ({ payload }) => {
         // ...
     },
+    // Opt-in to enable compatibility with
+    // Amazon SQS Extended Client Java Library (and other compatible libraries).
+    // see https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-s3-messages.html
+    extendedLibraryCompatibility: boolean;
```

Comment:
We need to add this option to the README for `sqsProducer` as well, right?

Reply:
I think so, added.